### DISCUSSION TITLE
#100DaysOfGatsby - Challenge 1 Complete

### DISCUSSION THREAD

**Discussion by Kevpedia** [gatsbyjs/gatsby#28810] [Community]
here we are! https://audioc0remain38977.gtsb.io/

**Comment by natac13** [comment]
@Kevpedia where did you find the 3 location?

**Reply by hashimwarren** [reply]
You can choose your own cities. But we maybe should be more explicit. Thanks for that question

**Comment by Kevpedia** [comment]
I didn't, so I just made them up by picking 3 random CA cities

**Comment by natac13** [comment]
Here is mine https://build-2d8ba79f-8862-4848-b385-0407784680c7.gtsb.io/

**Comment by marieqg** [comment]
Hello, Here is mine: https://build-01055a0d-9f0d-46b3-8257-99c03a2bd525.gtsb.io/

**Comment by SumanKisku** [comment]
Here is my preview but I have built all the pages manually - https://preview-audioc0remain84133.gtsb.io/

**Reply by natac13** [reply]
Do yo you help with the file system routing? https://www.gatsbyjs.com/docs/reference/routing/creating-routes/#using-the-file-system-route-api

**Comment by meaganewaller** [comment]
Got mine up today! https://audioc0re13944.gtsb.io/

**Comment by ramojej** [comment]
Hi, everyone! Here's mine: https://build-8ddc2bbd-7708-4f25-8941-ba2ee655a543.gtsb.io/ 

I used WordPress as my CMS though. I hope it's fine.

**Comment by Jokzin** [comment]
Hi,

Here is mine : https://audioc0re76897.gtsb.io/

Only location are sourced from Contenfull. (i'll update to sourced every content from Contentfull later)

**Comment by analia-mok** [comment]
Hiya, 
Completed my preview today: https://audioc0reanaliamok.gtsb.io/ 🎉

**Comment by mlisdev** [comment]
I kind of can't believe I got it done today. 👩🏼‍💻
https://audioc0remaster30709.gtsb.io/

**Comment by Tonye123** [comment]
Hello! 
This is mine - https://build-6176f1c2-8307-41e9-afc2-65630c3b61e8.gtsb.io

**Comment by orgabor** [comment]
Hi all,
this is my site:
https://audioc0reorgabor.gtsb.io/

**Comment by jatne** [comment]
Hello everyone! 
[https://jatne_audioc0re.gtsb.io/](https://jatne_audioc0re.gtsb.io/)

**Comment by Sterling-Smith** [comment]
https://build-b7a7c9fe-6234-4314-ba46-d28036069ac8.gtsb.io/ <<-- Done!

**Comment by wad-im** [comment]
Hey everyone, 
here comes my prototype [https://100daysofgatsbyaudiocore.gtsb.io/](https://100daysofgatsbyaudiocore.gtsb.io/)

**Comment by VernerKeel** [comment]
All right, mr. Senior Dev, [click here to see the Audi0Core POC you requested ](https://audiocore.webcontext.uk).
Pages are created using **Gatsby File System Route API**, all the content is sourced from **Contentful** & the site is builded/deployed on **Gatsby Cloud.** 

I stumbled upon on a few issues, however. After installing dependencies the system found **8 high severity vulnerabilities** which could not be fixed using npm audit fix. Some vulnerabilities were referring to 'axios' packages 'server-side request forgery' issue & mentioned that this issue is fixed in axios@0.21.1 version but installing it did not resolve the situation. The rest of vulnerabilities pointed to 'prototype-pollution' in 'utilis-extend package' used by 'gatsby-source-contentful'  When I read about this issue from npmjs.com then it turns out that there's no fix available. 

Does that mean that I should just ignore those vulnerabilities or is there anything I could do to fix this stuff?

**Comment by tdudkowski** [comment]
Not completed yet. But I'm here to ring a bell at the time (still 8th of January?).
Basic settings now, and still I must to learn that File System Route API and Contentful data.
[build-268bb86a-85c2-425f-88bf-214aa6c10eaf.gtsb.io](https://build-268bb86a-85c2-425f-88bf-214aa6c10eaf.gtsb.io/)

**Reply by tdudkowski** [reply]
In the last moment but done. I did it both ways for /locations - gatsby-node, and, the second one File System Route API - [https://challengegatsby100.gtsb.io/](https://challengegatsby100.gtsb.io/)

**Comment by Kevpedia** [comment]
@hashimwarren - did you send out challenge 2 today? I didn't receive any email

**Reply by Kevpedia** [reply]
nevermind! just noticed this line:
> BRIEFING 2 – January 22 – Just like in a real startup, one sprint ends and another begins! Visit the Gatsby blog to learn about the next challenge!

**Comment by BohdanOne** [comment]
Hi all 👋,
here is my solution https://audioc0remain89106.gtsb.io

**Reply by natac13** [reply]
Nice Picture!

**Reply by BohdanOne** [reply]
It's from unsplash, you can find link to original in the footer on my site

**Comment by JRod** [comment]
Here my PoC https://sharetone.gtsb.io/

**Comment by maarjaroots** [comment]
Hello everybody 👋, my [AudioC0re PoC](https://audio.webcontext.uk/)

**Reply by natac13** [reply]
Could I ask what PoC means?

**Reply by maarjaroots** [reply]
Proof of concept

**Reply by natac13** [reply]
Thank you. Too many acronyms for me to keep up with theses days.

**Comment by rishikalokesh2512** [comment]
Hi all,
my solution for challenge 1- https://audiocore29782.gtsb.io/

**Comment by inakicalvo** [comment]
Hi folks!

Here's my solution for the first sprint: https://audiocoremain20023.gtsb.io/

This was my first time using the File System Route API and Contentful.

I thought it would be much harder, but in fact it wasn't. I've really enjoyed building this Gatsby site!

I'm looking forward to the next sprint.

**Comment by Aereli** [comment]
Hi All, I’m am glad to present you my solution to challenge one.
I used [Sanity.io](https://create.sanity.io/) as the headless CMS and [Netlify](https://www.netlify.com/) to deploy.
[project](https://100daysofgatsby2021.netlify.app/) 🤛  🏄

**Comment by tdudkowski** [comment]
Here's my solution https://challengegatsby100.gtsb.io/
I did it both ways for /locations page: gatsby-node and File System Route
API, still not fully grasping how it works, but it works, and in a damn
good way.

pt., 8 sty 2021 o 23:24 Kevin <notifications@github.com> napisał(a):

**Comment by jurek2006** [comment]
Here is my solution https://100daysofcodejurekskowron.gtsb.io/

I used the File System Route API to generate also /home and /about (as well as locations) based on 'page' schema defined in contentful. But I haven't found **how to redirect the index page (/) to /home (generated with FSR API).** Redirect mentioned [here](https://stackoverflow.com/questions/54870485/how-to-redirect-from-the-index-page-to-a-route-which-is-generated-programmatical) doesn't work in this case (It changes the URL to /home but the page doesn't exist then and it gives 404. Although we can manually open /home and it exists).
I have found a solution for redirecting with gatsby-node - but I'm pretty sure this works only when we generate pages in gatsby-node, not with FSR API).
[I'm aware of the fact, that redirecting / to /home which is generated from content entry is probably a very bad idea - when an editor deletes the Home entry in 'pages' schema, the whole page won't work. I was rather interested if it is possible to redirect to a page generated with FSR API]

**Comment by davik4life** [comment]
Hi everyone, got in a little late, but I have completed my Challenge 1 Here's the link: https://preview-gatsby100dayschallengemain.gtsb.io/

Unto the next one, Happy Coding.
